community/kernel/src/main/java/org/neo4j/kernel/impl/index/labelscan/NativeLabelScanStore.java

```diff
@@ -95,6 +96,12 @@
     private static final byte REBUILDING = (byte) 0x01;
 
     /**
+     * Native index monitor tag, to distinguish monitors for native index store
+     * from others.
+     */
+    public static final String NATIVE_INDEX_TAG = GraphDatabaseSettings.LabelIndex.NATIVE.name();
```

Comment:
Can we be a little bit more explicit about what we call this tag to not confuse it with "native schema index"? My suggestion is something like: `NATIVE_LABEL_INDEX_TAG`.
Let this also be reflected in javadoc:
- "Native index monitor tag" -> "Native label index monitor tag"
- "monitors for native index store" -> "monitors for native label index"